tests/cypress/support/accessibility.js

```diff
@@ -1,3 +1,9 @@
+const a11yOptions = {
+  runOnly: {
+    type: 'tag',
+    values: ['wcag21aa']
```

Comment:
Axe nowadays supports `wcag22aa` as well, shouldn't we use that?

Reply:
The idea of using WCAG 2.2 rules is good, but if it is not certain whether such a change might cause additional work compared to aiming to the current level of rules, it is better to leave this potentially for the next iteration. The University of Helsinki requires a level of accessibility that meets WCAG 2.1 AA, and the changes made now fulfill this requirement. In summary: a good idea and worth implementing, but better to do it later.